package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "internetconnectionchecker",
+  "version": "1.0.0",
+  "description": "This tool allows you to make a costant checks on your Internet connection using the Speedtest.net API. I'm sorry if the code is not beautiful but I've made this just for myself in less than a day, however, it should work correctly.",
+  "main": "app.js",
+  "scripts": {
```

Comment:
i cloned this fork and ran `npm start` out of habit. might be worth adding here!

```js
"start": "node app.js"
```